peripherals/spi/spi_v3.yaml

```diff
@@ -0,0 +1,281 @@
+      NotMasked: [1, "Overrun interrupt not masked"]
+    UDRIE:
+      Masked: [0, "Underrun interrupt masked"]
+      NotMasked: [1, "Underrun interrupt not masked"]
+    TXTFIE:
+      Masked: [0, "Transmission transfer filled interrupt masked"]
+      NotMasked: [1, "Transmission transfer filled interrupt not masked"]
+    EOTIE:
+      Masked: [0, "End-of-transfer interrupt masked"]
+      NotMasked: [1, "End-of-transfer interrupt not masked"]
+    DPXPIE:
+      Masked: [0, "Duplex transfer complete interrupt masked"]
+      NotMasked: [1, "Duplex transfer complete interrupt not masked"]
+    TXPIE:
+      Masked: [0, "TX space available interrupt masked"]
+      NotMasked: [1, "TX space available interrupt not masked"]
+    RXPIE:
+      Masked: [0, "RX data available interrupt masked"]
+      NotMasked: [1, "RX data available interrupt not masked"]
+
+  SR:
+    CTSIZE: [0, 65535]
+    RXWNE:
+      LessThan32: [0, "Less than 32-bit data frame received"]
+      AtLeast32: [1, "At least 32-bit data frame received"]
+    RXPLVL:
+      ZeroFrames: [0, "Zero frames beyond packing ratio available"]
+      OneFrame: [1, "One frame beyond packing ratio available"]
+      TwoFrames: [2, "Two frame beyond packing ratio available"]
+      ThreeFrames: [3, "Three frame beyond packing ratio available"]
+    TXC:
+      Ongoing: [0, "Transmission ongoing"]
+      Completed: [1, "Transmission completed"]
+    SUSP:
+      NotSuspended: [0, "Master not suspended"]
+      Suspended: [1, "Master suspended"]
+    TSERF:
+      NotLoaded: [0, "Additional number of SPI data to be transacted not yet loaded"]
+      Loaded: [1, "Additional number of SPI data to be transacted was reloaded"]
+    MODF:
+      NoFault: [0, "No mode fault detected"]
+      Fault: [1, "Mode fault detected"]
+    TIFRE:
+      NoError: [0, "TI frame format error detected"]
+      Error: [1, "TI frame format error detected"]
+    CRCE:
+      Match: [0, "No CRC error detected"]
+      NoMatch: [1, "CRC error detected"]
+    OVR:
+      NoOverrun: [0, "No overrun occurred"]
+      Overrun: [1, "Overrun occurred"]
+    UDR:
+      NoUnderrun: [0, "No underrun occurred"]
+      Underrun: [1, "Underrun occurred"]
+    TXTF:
+      NotCompleted: [0, "Transmission buffer incomplete"]
+      Completed: [1, "Transmission buffer filled with at least one transfer"]
+    EOT:
+      NotCompleted: [0, "Transfer ongoing or not started"]
+      Completed: [1, "Transfer complete"]
+    DXP:
+      Unavailable: [0, "Duplex packet unavailable: no space for transmission and/or no data received"]
+      Available: [1, "Duplex packet available: space for transmission and data received"]
+    TXP:
+      Full: [0, "Tx buffer full"]
+      NotFull: [1, "Tx buffer not full"]
+    RXP:
+      Empty: [0, "Rx buffer empty"]
+      NotEmpty: [1, "Rx buffer not empty"]
+
+  IFCR:
+    "*C":
+      _write:
+        Clear: [1, "Clear interrupt flag"]
+
+  TXDR:
+    TXDR: [0, 4294967295]
```

Comment:
This and `CRCPOLY` might be clearer as `0xFFFFFFFF`

Reply:
ACK